packages/cli/src/validator/validate-locales.ts

```diff
@@ -0,0 +1,48 @@
+import chalk from 'chalk';
+import meow from 'meow';
+
+import jsonDiff from 'json-diff';
+
+import nl from '../../../app/src/locale/nl.json';
+import en from '../../../app/src/locale/en.json';
+
+function objectsHaveSameKeys(
+  obj1: Record<string, unknown>,
```

Comment:
Can we perhaps get rid of the `unknown`?

```suggestion
  obj1: Record<string, LocaleValue>,
```

if you define `type LocaleValue = string | Record<string, LocaleValue>;` or something similar.

Reply:
I don't know if there are still arrays in the locale file, but then they would have to be included as well I guess...